### DISCUSSION TITLE
ColorPicker组件api、demo优化讨论

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#90] [General]
# api

1. 自定义弹出颜色面板方式（通过按钮等，默认情况下ColorPicker只是选择颜色，不带弹出效果）
2. 面板内部内容自定义组装（基础、高级、历史等）、颜色小块组件化
3. 展示颜色渐变更美观

**Comment by ErKeLost** [comment]
API 问题：
一，color-Picker 预定义Api
1. v-model 绑定Value值 支持输入rgb hsl hex hsv
2. disabled 禁用选择器
3. size 选择器大小
4. showAlpha 是否展示alphaSlider
5. showHistory 是否展示历史颜色
6. showPalette 是否展示调色板
7. showPredefine（预定义颜色） | showSwatches（颜色样本） 是否展示 预定义样本色板
8. dotSize 调色板圆点大小
9. swatches 自定义样本色板
10. mode 切换不同模式
11. historyStorage 历史颜色是否存储到localstorage

功能优化：
 一，TODO 是否 需要支持渐变色 是否影响整体改动
 二，TODO 自定义弹出方式  通过按钮 等
————————————————————————————
(tips): 如果有对API ,功能 建议 感谢评论
